scripts/linters/python_linter.py

```diff
@@ -317,6 +338,30 @@ def all_filepaths(self):
         """Return all filepaths."""
         return self.files_to_lint
 
+    def _get_trimmed_error_messages(self, lint_messages):
+        """Remove extra bits from pylint messages.
+
+        Args:
+            lint_messages: list. Messages returned by the python linter.
+
+        Returns:
+            str. A string with the trimmed messages.
+        """
+        error_messages = ''
+        # Remove newlines and coverage report from the end of message.
+        lint_messages = lint_messages[:-5]
+        for message in lint_messages:
+            python_utils.PRINT(message)
```

Comment:
Why print messages? (I don't expect this function to print something)

Reply:
We need to print message-ids as well that's why we need it. This line will print full messages.

Reply:
Then do it in the method which is calling this why printing through a` _get_` message?

Reply:
Done.